CodeEdit.xcodeproj/project.pbxproj

```diff
@@ -673,8 +674,21 @@
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"CodeEdit/Preview Content\"";
 				DEVELOPMENT_TEAM = "";
+				ENABLE_APP_SANDBOX = YES;
```

Comment:
I seriously doubt we want to enable App Sandbox; doing so will break most of the app. Was this automatically added by Xcode?